docs/release_notes.rst

```diff
@@ -0,0 +1,207 @@
+=======================================================================================
+PUDL Release Notes
+=======================================================================================
+
+---------------------------------------------------------------------------------------
+0.4.0 (2021-07-XX)
+---------------------------------------------------------------------------------------
+This is a ridiculously large update including more than a year and a half's
+worth of work. We're going to start doing much more frequent incremental
```

Comment:
it is my understanding that is it best practice to not say "we will be doing ____"

Reply:
Haha, very true.

Reply:
exactly. I'd suggest removing this... any day now.

Reply:
Will do.